### PR TITLE
Added Cart

### PR DESCRIPTION
**Pull request by HrishikeshMahalle**
Cart page!
Screenshot (668)

### CODE REVIEW

**Comment by agrawalshanu708** [comment]
Try to consistent with colors, my suggestion to make background color light..and else is looking good

**Comment by HrishikeshMahalle** [comment]
> Try to consistent with colors, my suggestion to make background color light..and else is looking good



> @agrawalshanu708  Sure i'll look into it, initially thought of making the website theme darker but the color seletion went quite terrible so will try to update that once Merged. Thank you

**Review by AshutoshM29 — CHANGES_REQUESTED** [review]
UI looks good!
Just a suggestion, you are using too many high-quality images which are taking a lot of time to load your website, you should optimize your images!!

> @AshutoshM29 Will surely work over that as well, since i'll add more images, will keep a note of this.. Thank You!!!!

**Review by AshutoshM29 — APPROVED** [review]

File: Cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+}
+
+.header .header-title{
+    width: 40%;
+    text-align: center;
+    font-size: 2rem;
+    font-weight: 1000;
+}
+.header .header-title span{
+    background-color: #f3ec78;
```
Comment (md-amir-gauhar) [inline Cart/cart.css:61]: Try using variables for colors

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+.head-container{
+    position:relative; 
+    width: 100%;
+    height: 45rem;
+    border: solid;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* background:linear-gradient(transperant, black, 30) url("./Assets/dying-light-2.jfif");
+    background-repeat: no-repeat;
```
Comment (md-amir-gauhar) [inline styles.css:22]: Delete unused/commented codes

**Review by md-amir-gauhar — APPROVED** [review]

**Review by Hamza-777 — APPROVED** [review]
Everything looks good! Just make the changes amir suggested.

**Pull request merged**